.eslintrc.js

```diff
@@ -1,52 +1,36 @@
 module.exports = {
-    "env": {
-        "browser": true,
-        "es6": true,
-        "jquery": true,
-        "jasmine": true
+    env: {
+        'jquery': true,
+        'jasmine': true
     },
-    "extends": "eslint:recommended",
-    "rules": {
-        "no-global-assign": 2,
-        "indent": [
-            2,
-            4
-        ],
-        "linebreak-style": [
-            2,
-            "unix"
-        ],
-        "quotes": [
-            2,
-            "single"
-        ],
-        "semi": [
-            2,
-            "always"
-        ],
-        "curly": [
-            2,
-            "all"
-        ],
-        "camelcase": [
-            2,
-            {
-                "properties": "always"
-            }
-        ],
-        "eqeqeq": [
-            2,
-            "smart"
-        ],
-        "one-var-declaration-per-line": [
-            2,
-            "always"
-        ],
-        "new-cap": 2
+    extends: '@mozilla-protocol/eslint-config',
+    rules: {
+        'no-useless-escape': 1,
```

Comment:
Note for reviewer: I didn't go and fix all the regular expressions this rule picked up on, as it would have required a lot of manual testing. I set this to a warning instead, and we can work on eliminating these over time.